test/integration/infrastructure/infrastructure_test.go

```diff
@@ -149,61 +151,143 @@ var _ = Describe("Infrastructure tests", func() {
 	})
 
 	Context("with infrastructure that requests existing vpc", func() {
-		var (
-			identifiers infrastructureIdentifiers
-		)
+		It("should successfully create and delete", func() {
+			identifiers := prepareVPC(ctx, clientFactory, *region, vpcCIDR, natGatewayCIDR)
```

Comment:
we could simply puth vpc as a suite scpe resouce. so  you may put it in BeforeSuite.